README.md

```diff
@@ -34,6 +34,8 @@ const path = require('path')
 fastify.register(require('@fastify/secure-session'), {
   // the name of the session cookie, defaults to 'session'
   cookieName: 'my-session-cookie',
+  // the name of the attribute decorated on the request-object, defaults to 'session'
+  fieldName: 'my-session',
```

Comment:
We don't have a "standard" for it.

`name` is too generic IMHO

`fastify-env` uses `confKey`, maybe `sessionKey` could fit better?
https://github.com/fastify/fastify-env#usage